providers/attio/write.go

```diff
@@ -55,8 +46,40 @@ func (c *Connector) Write(ctx context.Context, config common.WriteParams) (*comm
 	return constructWriteResult(config.ObjectName, body)
 }
 
+func (c *Connector) buildWriteURL(config common.WriteParams) (*urlbuilder.URL, error) {
+	if supportAttioGeneralApiWrite.Has(config.ObjectName) {
+		return c.getApiURL(config.ObjectName)
+	}
+
+	url, err := c.getObjectWriteURL(config.ObjectName)
+	if err != nil {
+		return nil, err
+	}
+
+	return url, nil
+}
+
+// determineWriteMethod selects the appropriate HTTP method based on config.
+func (c *Connector) determineWriteMethod(config common.WriteParams) common.WriteMethod {
```

Comment:
This is subjective, but I would prefer to see this in the main body in line with other connectors